src/lib.native.ts

```diff
@@ -855,9 +855,9 @@ export default {
   crypto_secretbox_keygen,
   crypto_secretbox_NONCEBYTES,
   crypto_secretbox_open_easy,
+  crypto_sign_detached,
   crypto_sign_keypair,
   crypto_sign_seed_keypair,
-  crypto_sign_detached,
```

Comment:
this is a nit, but... I moved that `crypto_sign_detached` down several positions because that matches the order it appears in the source.

Reply:
I made that change to follow the order; I'm not sure what you mean. Your change below shows it out of order : https://github.com/serenity-kit/react-native-libsodium/compare/main...getify:react-native-libsodium:main 
![image](https://github.com/getify/react-native-libsodium/assets/8634344/0c3fa5a6-dc3d-4e83-9951-6bbe9c088e8d)

The export is alphabetical and i left crypto_sign_ed25519_pk_to_curve25519 stay out of order so as not to nit

Reply:
I was trying to re-order the exports (at least the ones I was touching, or near to) in the same order as the function definitions themselves appear in the source. moreover I put the ed25519* function definitions at the end of the "crypto_sign_*" set of functions.

in any case, I didn't recognize the alphabetical ordering of exports (that's not something I ever do personally), so... yeah, just ignore my nit. re-arrange the exports however makes sense for the project's conventions.